### ISSUE TITLE
Cef Sharp.wpf Issue with windows 10

### ISSUE THREAD

**Issue by nextolive2016** [cefsharp/CefSharp#1779]
` <wpf:ChromiumWebBrowser Name="ChromiumWebBrowser1"   Address="google.com"></wpf:ChromiumWebBrowser>`
When we are running my application on the window 10 then it's refreshing or (glitching) after 2 or 3 second. We are using CefSharp.Wpf 49.0.1 Version. And also we are using modern ui in my project so when we press backspace key then its sending on previous user control. Can I fix this?
 [See Vedio](https://www.dropbox.com/s/pspru09a9nopg1u/Video%20Jul%2027%2C%2010%2000%2022%20AM.mov?dl=0)

**Comment by amaitland** [comment]
See the release notes for known issues.

**Comment by nextolive2016** [comment]
can you  summarized here and please pass link for us.

**Comment by amaitland** [comment]
> what is mention in release notes? Can you summarized here and please pass link for us.

Read the release notes, they're linked above under `Releases`. It's very important that you read them.

**Comment by nextolive2016** [comment]
Hi, I have read this blog but how to disable GPU in my application. Please share we are facing lots of  problem.

**Comment by amaitland** [comment]
@nextolive2016 There are **many many** references to disabling `GPU` acceleration. Search above and I'm sure you'll find them.

https://github.com/cefsharp/CefSharp/releases
